pkg/docker/client.js

```diff
@@ -407,7 +407,7 @@
                 $(self).off("info", got_info);
```

Comment:
All related references to ```$(xxx).off(...)``` need to be done too. And in some cases things will need to be refactored to use named and functions that ```removeEventListener``` can refer to.